src/components/BlacklistBlock/index.tsx

```diff
@@ -78,22 +79,33 @@ const BlacklistBlock = ({
       <div className="flex items-center justify-between">
         <div className="mr-6 min-w-0 flex-1 flex-col items-center text-sm leading-5">
           <div className="white mb-1 flex flex-nowrap">
-            <Tooltip content={intl.formatMessage(messages.blacklistedby)}>
-              <UserIcon className="mr-1.5 h-5 w-5 min-w-0 flex-shrink-0" />
-            </Tooltip>
-            <span className="w-40 truncate md:w-auto">
-              <Link
-                href={
-                  data.user.id === user?.id
-                    ? '/profile'
-                    : `/users/${data.user.id}`
-                }
-              >
-                <span className="font-semibold text-gray-100 transition duration-300 hover:text-white hover:underline">
-                  {data.user.displayName}
+            {data.user != null ? (
```

Comment:
Please use `!==`

Reply:
I use != here to check for both null and undefined, instead of only null. Things would similarly break if the user was undefined.

Reply:
Updated: became `{data.user ? (`